cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -7,47 +7,68 @@
 
 from cfme import test_requirements
 from cfme.cloud.provider.ec2 import EC2Provider
+from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.markers.env_markers.provider import ONE_PER_TYPE
 from cfme.services.myservice import MyService
 from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.blockers import BZ
 from cfme.utils.update import update
 from cfme.utils.wait import wait_for
 
 
 pytestmark = [
     pytest.mark.long_running,
     pytest.mark.ignore_stream("upstream"),
+    pytest.mark.meta(blockers=[BZ(1677548, forced_streams=["5.11"])]),
```

Comment:
Is there a workaround so that we can still run this tests, since this BZ is a RFE? I have the same question on #8977

Reply:
@john-dupuy There is no workaround present atm.

Reply:
This BZ is actually an RFE, and this functionality is under development phase. Not fully implemented on CFME 5.11. Once it is available on 5.11 I will remove BZ blocker from there.